playground/src/containers/routes/index.tsx

```diff
@@ -22,14 +21,14 @@ const Configuration = loadable(async () => import("pages/condiguration"), asyncO
 
 export const Routes = observer(() => {
   return (
-    <Router history={history}>
+    <BrowserRouter>
```

Comment:
I think we can't use BrowserRouter here, since we need to synchronize history with mobx 🤔